api/validation/schemas.py

```diff
@@ -1,21 +1,19 @@
 from marshmallow import Schema, fields, validate, INCLUDE
 
 from api.validation.validators import comma_splittable, aws_region_validator, is_alphanumeric_with_hyphen, valid_api_log_levels_predicate
-from api.logging import VALID_LOG_LEVELS
-
 
 class EC2ActionSchema(Schema):
     action = fields.String(required=True, validate=validate.OneOf(['stop_instances', 'start_instances']))
-    instance_ids = fields.String(required=True, validate=comma_splittable)
+    instance_ids = fields.String(required=True, validate=validate.And(comma_splittable, validate.Length(max=2048)))
```

Comment:
suggestion: even though the endpoint can receive a list of ids, the UI is configured to operate on a single instance a t a time. I suggest to limit this field to the length of an instance id, and, if you want to do the extra mile, make this endpoint accept a single instance instead of an array

Reply:
Nice one! I'll do it gladly, but I would do this in a new PR so the endpoint (and its validation schema) get changed in the same PR context